src/main/java/teammates/common/util/Sanitizer.java

```diff
@@ -47,23 +47,23 @@ public static String sanitizeEmail(String rawEmail) {
     }    
     
     /**
-     * Sanitizes a Instructor or Student's name by removing leading/trailing whitespace.
+     * Sanitizes a Instructor or Student's name by removing leading, trailing, and duplicate internal whitespace.
```

Comment:
No need to specify how we use the function so `Sanitizes name by removing leading, trailing, and duplicate internal whitespace.` should be good enough.